io.github.zaps166.QMPlay2.Devel.json

```diff
@@ -0,0 +1,156 @@
+{
+  "app-id": "io.github.zaps166.QMPlay2",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.6",
+  "sdk": "org.kde.Sdk",
+  "command": "QMPlay2",
+  "rename-icon": "QMPlay2",
+  "rename-desktop-file": "QMPlay2.desktop",
+  "rename-appdata-file": "QMPlay2.appdata.xml",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--filesystem=home",
+    "--filesystem=/run/media",
+    "--filesystem=/media",
+    "--persist=.qmplay2",
+    "--device=dri",
+    "--system-talk-name=org.freedesktop.UDisks2"
+  ],
+  "modules": [
+   {
+      "name": "libass",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--enable-harfbuzz",
+        "--enable-fontconfig"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/libass/libass/",
+          "tag": "0.17.0",
+          "commit": "bef4b43ef1882b77f789f611d9cd24271ccdd65b"
+        },
+        {
+          "type": "script",
+          "commands": [
+            "autoreconf -fiv"
+          ],
+          "dest-filename": "autogen.sh"
+        }
+      ]
+    },
+    {
+      "name": "libva",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/intel/libva/",
+          "tag": "2.16.0",
+          "commit": "1333034d7ec6e4f8bcb340d8f7b81b8d32835c49"
+        }
+      ]
+    },
+    {
+      "name": "gme",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://bitbucket.org/mpyne/game-music-emu",
```

Comment:
New URL: https://github.com/libgme/game-music-emu

It's also good to cherry-pick the `414e0d993548fa6e1aea8b944da531e7db07bb94`.